--- a/iroh-blobs/src/api/blobs/export_pairs.rs
+++ b/iroh-blobs/src/api/blobs/export_pairs.rs
@@ -8,7 +8,6 @@
     Poseidon2Backend, CHUNK_SIZE,
 };
 use n0_future::StreamExt;
-use support::*;
 
 use super::{EncodedItem, ExportBaoProgress};
 use crate::{
@@ -16,6 +15,7 @@
     provider::events::ProgressError,
     store::{readonly_mem::ReadonlyMemStore, IROH_BLOCK_SIZE},
 };
+use support::*;
 
 async fn chunks(export: ExportBaoProgress) -> Vec<Vec<u8>> {
     let mut stream = export.into_byte_stream();
